UsageCollectorWorkerService/Models/RootSysResUsageValues.cs

```diff
@@ -0,0 +1,6 @@
+namespace UsageCollectorWorkerService.Models;
+
+public class RootSysResUsageValues
```

Comment:
This model might be unnecessary. You can probably just use List<SysResUsageValues> everywhere. But feel free to keep it, if you prefer this approach.